src/types.ts

```diff
@@ -31,23 +31,15 @@ export interface SpendFailure {
   errorCode?: string
 }
 
-export interface SynchronizerStatus {
+export interface StatusEvent {
   alias: string
   name:
     | 'STOPPED' /** Indicates that [stop] has been called on this Synchronizer and it will no longer be used. */
     | 'DISCONNECTED' /** Indicates that this Synchronizer is disconnected from its lightwalletd server. When set, a UI element may want to turn red. */
-    | 'DOWNLOADING' /** Indicates that this Synchronizer is actively downloading new blocks from the server. */
-    | 'VALIDATING' /** Indicates that this Synchronizer is actively validating new blocks that were downloaded from the server. Blocks need to be verified before they are scanned. This confirms that each block is chain-sequential, thereby detecting missing blocks and reorgs. */
-    | 'SCANNING' /** Indicates that this Synchronizer is actively decrypting new blocks that were downloaded from the server. */
-    | 'ENHANCING' /** Indicates that this Synchronizer is actively enhancing newly scanned blocks with additional transaction details, fetched from the server. */
```

Comment:
We don't want these event name types?

Reply:
Older versions of the SDK used them but they don't exist anymore